references/recognition/train_pytorch.py

```diff
@@ -210,7 +229,8 @@ def parse_args():
     parser = argparse.ArgumentParser(description='DocTR train text-recognition model (PyTorch)',
                                      formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
-    parser.add_argument('data_path', type=str, help='path to data folder')
+    parser.add_argument('train_data_path', type=List[str], help='list of path(s) to train data folder')
```

Comment:
Are you positive that we're able to pass list of string to a bash script?

Reply:
Perhaps keeping the same old path, and simply assuming that all parts of the training data are inside that folder?